conf/nautilus/ansible/tier_2_deploy.yaml

```diff
@@ -0,0 +1,44 @@
+globals:
```

Comment:
Could we rename this file to `tier-2_deploy.yaml`? This would fall into the new guidelines for naming the files. Sorry for the late push